pkgs/development/python-modules/findpython/default.nix

```diff
@@ -13,19 +13,16 @@
 , pytestCheckHook
 }:
 
-let
+buildPythonPackage rec {
   pname = "findpython";
-  version = "0.1.6";
-in
-buildPythonPackage {
-  inherit pname version;
```

Comment:
What was the problem with the let binding that you felt you needed to change that?

Reply:
tbh i didn't think to much about it. It's just a little cleaner and the way it seems to mostly be done.

Reply:
seems like that issue resolved itself anyhow, since the findpython and pdm-pep517 updates have already found their way onto master